src/evm-block-extractor/src/database/mod.rs

```diff
@@ -92,4 +92,22 @@ pub trait DatabaseClient: Send + Sync {
 
     /// Get earliest block number
     async fn get_earliest_block_number(&self) -> anyhow::Result<u64>;
+
+    /// Delete latest blocks starting with `start_from`, and related transactions.
+    /// Deleted blocks and transactions will be preserved in special table with
```

Comment:
why is it a "Special" table? Write instead the name of the table

Reply:
Replaced word 'special' with word 'discarded'. Since it is a documentation on trait, it will not be convenient to pre-define specific DB table on abstraction level.